crates/cxx-qt-lib/src/core/qlinef.rs

```diff
@@ -117,6 +153,14 @@ impl From<&ffi::QLine> for QLineF {
     }
 }
 
+impl From<QLineF> for ffi::QLine {
```

Comment:
wonder why we did `QPointF` and not `&QPointF` before as this creates a copy anyway. Maybe it just felt more Rust like to do by-value. We have a load of places where we need to decide on by-ref or by-value, cc @LeonMatthesKDAB , this is another one.